content/zh/blog/2023/istiocon-china/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "IstioCon China 2023 总结"
+description: 简要回顾上海 KubeCon + CloudNativeCon + Open Source Summit China 中的 Istio 主题演讲。
+publishdate: 2023-09-29
+attribution: "IstioCon 2023 中国站程序委员会; Translated by Wilson Wu (DaoCloud)"
+keywords: [Istio Day,IstioCon,Istio,conference,KubeCon,CloudNativeCon]
+---
+
+经过两年时间只举办虚拟活动后，能够再次线下欢聚一堂真是太好了。
+我们的活动已经排满了 2023 年的日历。
+[Istio Day 欧洲站](/zh/blog/2023/istio-at-kubecon-eu/)于 4 月举行，
+[Istio Day 北美站](https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/co-located-events/istio-day/)将于今年 11 月举行。
+
+IstioCon 致力于打造行业领先的服务网格，
+提供一个平台来探索从现实世界的 Istio 部署中获得的见解、
+参与交互式实践活动，并与整个 Istio 生态系统的维护者进行交流。
+
+除了 [IstioCon 2023 虚拟](https://events.istio.io/)活动之外，
+[IstioCon 2023 中国站](https://www.lfasiallc.com/kubecon-cloudnativecon-open-source-summit-china/co-located-events/istiocon-cn/)也于 9 月 26 日在中国上海举行。
+该活动是 KubeCon + CloudNativeCon + Open Source Summit China 的一部分，
+由 Istio 维护者和 CNCF 安排和主办。
+我们非常自豪能够在上海举办如此盛大的 IstioCon 活动，
+并很高兴将中国 Istio 社区的成员聚集在一起。
+此次活动证明了 Istio 在亚太生态系统中巨大的受欢迎程度。
+
+{{< image link="./group-pic.jpg"
+    caption="IstioCon 2023 中国站"
+    >}}
+
+IstioCon 中国站在程序委员会成员宋净超和徐中虎的开幕主题演讲中拉开帷幕。
+此次活动内容丰富，从新功能到最终用户演讲，
+主要关注新的 Istio Ambient 网格。
+
+{{< image width="75%"
+    link="./opening-keynote.jpg"
+    caption="欢迎来到 IstioCon 2023 中国站"
+    >}}
+
+欢迎致辞之后，来自 Google 的 Justin Pettit 发表了主题演讲，
+主题为“Istio Ambient Mesh 作为托管基础设施”，
+强调了 Ambient 模式在 Istio 社区中的重要性和优先级，
+特别是对于像 Google Cloud 这样的我们的顶级支持者。
+
+{{< image width="75%"
+    link="./sponsored-keynote-google.jpg"
+    caption="IstioCon 2023 中国站，Google Cloud 赞助的主题演讲"
+    >}}
+
+主题演讲结束后，英特尔的张怀龙和阿里巴巴的曾宇星讨论了
+Ambient 和 Sidecar 共存的配置：对于想要尝试新
```

Comment:
Hi @wilsonwu, there is a relative blog: https://istio.io/latest/blog/2023/traffic-for-ambient-and-sidecar/, is it possible to link here?

Reply:
Done, added the English version blog link, I will update it to the Chinese version when it available in website.